lightspeed/modules/con_overview-lightspeed-onpremise.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: CONCEPT
+
+[id="overview-lightspeed-onpremise_{context}"]
+
+= Overview
+
+This section provides information about the system requirements, prerequisites, and the process for setting up a {LightspeedshortName} on-premise deployment.
+
+== System requirements
+
+Your system must meet the following minimum system requirements to install and run the {LightspeedshortName} on-premise deployment.
+
+[cols="a,a",options="header"]
+|===
+|Requirement |Minimum requirement
+
+|RAM
+|5 GB
+
+|CPU
+|1
+
+|Local disk
+|40 GB
+|===
+
+To see the rest of the {PlatformName} system requirements, see _Chapter 4. System requirements_ in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_planning_guide/index#platform-system-requirements[{PlatformName} Planning Guide].
+
+[NOTE]
+====
+You must also have a trial or paid subscription to {ibmwatsonxcodeassistant}, and have a model in the {ibmwatsonxcodeassistant} deployment space. For information about creating a {ibmwatsonxcodeassistant} model, see the link:https://cloud.ibm.com/docs/watsonx-code-assistant[{ibmwatsonxcodeassistant} documentation].
+====
+
+== Prerequisites
+
+* You have administrator privileges to the {PlatformName}.
```

Comment:
I would change to "You have administrator privileges for RH AAP"

Reply:
Done